mesonbuild/dependencies/dub.py

```diff
@@ -328,7 +328,7 @@ def find_package_target(pkg: DubPackDesc) -> bool:
         for lib in bs['libs']:
             if os.name != 'nt':
                 # trying to add system libraries by pkg-config
-                pkgdep = PkgConfigDependency(lib, environment, {'required': 'true', 'silent': 'true'})
+                pkgdep = PkgConfigDependency(lib, environment, {'required': 'true', 'silent': True})
```

Comment:
From the use of the plural form in the commit message I have the impression that both of these were meant to be `True` and not `'true'`

Reply:
They are both supposed to be true... something happened when I rebased this